test/index.test.js

```diff
@@ -321,4 +343,36 @@ describe('module-resolver', () => {
             );
     });
   });
+
+  describe('no babelrc in path', () => {
+    const transformerOpts = {
+      babelrc: false,
+      plugins: [
+        [plugin, {
+          cwd: 'babelrc',
+        }],
+      ],
+    };
+    const { cwd } = process;
+
+    function mockCwd() {
+      return '/.fake';
+    }
+
+    beforeEach(() => {
+      process.cwd = mockCwd;
+    });
+
+    afterEach(() => {
+      process.cwd = cwd;
+    });
+
+    describe('should leave the path as is', () => {
+      testRequireImport(
+                './test/path',
```

Comment:
In this case the code that was not covered is being run anyway, but I'll make the example more clear.